tests/dummy/app/components/contrib-manager/template.hbs

```diff
@@ -0,0 +1,54 @@
+<h2> Add contributor </h2>
+<div class="form">
+    <div class="form-group">
+        <label> User ID </label> {{input value=newContributorId}}
+    </div>
+    <div class="form-group">
+        <label> Permissions: </label>
+        <div class="input-group">{{radio-button name="permission" value=READ checked=newContributorPermissions}} Read</div>
+        <div class="input-group">{{radio-button name="permission" value=WRITE checked=newContributorPermissions}} Read + Write</div>
+        <div class="input-group">{{radio-button name="permission" value=ADMIN checked=newContributorPermissions}} Administrator</div>
+    </div>
+    <div class="form-group">
+        <label> Bibliographic: </label>
+        <div class="input-group">{{radio-button name="bibliographic" value=true checked=newContributorIsBibliographic}} Bibliographic </div>
+        <div class="input-group">{{radio-button name="bibliographic" value=false checked=newContributorIsBibliographic}} Non-Bibliographic </div>
+    </div>
+    <div class="form-group">
+        <button {{action 'addContributor' newContributorId newContributorPermissions newContributorIsBibliographic}} class="btn btn-primary">Add</button>
+    </div>
+</div>
+
+<table class="table">
+    <tr>
+        <th>ID</th>
+        <th> Name </th>
+        <th> Permissions </th>
+        <th> Bibliographic </th>
+        <th> </th>
+    </tr>
+    {{#each contributors as |contrib|}}
+        <tr>
+            <td>
+                {{contrib.id}}
+            </td>
+            <td>
+                {{if contrib.unregisteredContributor contrib.unregisteredContributor contrib.users.fullName}}
+            </td>
+            <td>
+                <select onchange={{action 'permissionChange' contrib value='target.value' }}>
+                    <option value=READ selected={{if (eq contrib.permission READ) true false}}> Read </option>
+                    <option value=WRITE selected={{if (eq contrib.permission WRITE) true false}}> Read + Write </option>
+                    <option value=ADMIN selected={{if (eq contrib.permission ADMIN) true false}}> Administrator </option>
+                </select>
+            </td>
+            <td>
+                <input onchange={{action 'bibliographicChange' contrib value='target.value' }} checked=contrib.bibliographic}} type="checkbox" name="bibliographic">
```

Comment:
Note for separate audit after this PR: form controls should have associated label elements